src/main/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitChecker.java

```diff
@@ -96,12 +98,14 @@ public LocalChecker getChecker(@NonNull TaskListener listener, String apiUrl) {
             if (GitHubServerConfig.GITHUB_URL.equals(apiUrl)) {
                 // If the GitHub public API is being used, this will fallback to ThrottleOnOver
                 LocalChecker checker = ThrottleOnOver.getChecker(listener, apiUrl);
-                checker.writeLog("GitHub throttling is disabled, which is not allowed for public GitHub usage, so ThrottleOnOver will be used instead. To configure a different rate limiting strategy, go to \"GitHub API usage\" under \"Configure System\" in the Jenkins settings.");
+                checker.writeLog(
```

Comment:
should be aligned to `writeLog(` not `checker`.

basically are you using a fixed offset for wrapped indentation?  (ie 8 which is the lengh of `checker.` - in which case don't  use an offset that matches to the thing you are offsetting.

otherwise you can see something like

```
checkz(.writeLog(
            "a very long string),
            "and something else")
```

and that is worse than evil.

Reply:
@jtnord 
Same debatable point as above.   The indent you are saying is missing was (as far as I can tell) intentionally removed as redundant.

Reply:
@jtnord 
Regarding the align to column, you can change this the formatter as noted  [here](https://github.com/jenkinsci/github-branch-source-plugin/pull/392/files#r580691521) to be `82` and it will align to column, but it produces results that I think you'll like even less than the current results.  `50` is also an option, but it ends up even worse. 

But maybe I'm misunderstanding what you mean.

Reply:
so given the settings that are being used are offset by a specific amount (and not column) if it hard to see when something is wrapped in something like

```
foo(bar(wibble(baz, bob,
        jennygoestomarket),
        markstaysathome));
```

Because both `jennygoestomarket` and `markstaysathome` are `offset` by the same amount, and when scanning the code it looks like they can be the arguments to the same function (when they are not).
Whitespace is helpful to show what things apply to (which is why we indent blocks `{`)  to me the same applies here.
Maybe I misunderstand the settings you are applying? (but I have had many issues with eclipse when using this setting in the past)

disclaimer \#1  - I much prefer projects that are set to indent on column.
disclaimer \#2 I am mildly dyslexic and so this is important to me.  **My** brain scans and reads text in a specific and different to the vast majority of people.

now - I am not a major contributor to this project and so I am not trying to enforce my standards onto the people who do work with it much more than I do (I have only had a few minor contributions) and as such I will adapt when I need to work in this project.  However if this is the first step into trying to get something more standard into Jenkins ecosystem and rolling out more widely then being inclusive of all people (which includes the neuro diverse) should be taken into account, as descisions made here can make it more or less easy for people to contribute.

disclaimer \#3  on my projects when I remember to switch to *my* coding style I `indent` with tabs so that people can customize their indent view to support their needs (2 for some people 8 for others, and some have 4 or even 3) but `offset` with spaces.  this allows the code to look the same regardless of tab space setting and allow people to have a visual offset that conforms to the way they work best.

and finally disclaimer \#4 all attempts at auto formatting will break somewhere.  This is why I think auto formatters are evil.  let the bot do a formatting review, let a human decide to reject it's findings, or to easily make it better.  See the end part on disclaimer 2

Reply:
@jtnord 
1. The problem I've seen with with indenting to column is you end up with deeply indented code.   I assume your suggestion for addressing that is to refactor the code into smaller methods, avoiding deeply nested structures. Am I correct? 

2. Neuro-atypicalness and accessibility is certainly a valid concern.  I read slowly and also depend on formatting to allow me to take in code at a reasonable speed.  I definitely want to be sensitive to a broad range of contributors needs.  At the same time, consider that someone else might find  "align to column" harder to read due to its deeper indenting causing lines to wrap more often.  Which one wins? In the case of this change, I am looking to apply a standard formatting that most closely matches what is already being used. There are parts of this code base that seem to be using "align to column", but the majority of it does not.  

3. I believe this mix of tabs and spaces is referred to as "smart-tabs". It works best in "align to column" if I understand it correctly.  I'm not opposed to this, but as with point two it seems a larger change.   I've done like I did with [no-join](https://github.com/jenkinsci/github-branch-source-plugin/compare/master...bitwiseman:task/formatting-no-join?expand=1) and created a branch using this and "align to column" settings:  [bitwiseman:task:task/formatting...bitwiseman:task/formatting-columns](https://github.com/bitwiseman/github-branch-source-plugin/compare/task/formatting...bitwiseman:task/formatting-columns?expand=1).  Perhaps there's other settings that could be applied to do this better.  Perhaps you as the one bringing this up could try various setting to find one that is best in your opinion? 

4. I would instead say that **all attempts at formatting will be imperfect**, both humans and automated systems.  Auto-formatters are generally imperfect in rigid and predictable ways, whereas human formatters are imperfect in a wide range of random ways.  I categorically disagree that humans should be allowed to opt-out of auto-formatting - given the option humans will choose to do things their own way and then we end up talking about "better" or "worse" formatting instead of focusing on making the code better.

Reply:
My preference would be to use the autoformatter for things it can do reliably—sort imports, remove trailing whitespace, enforce spaces vs. tabs for block indentation, normalize use of spaces around / inside tokens like `(` `)` `=`—but for more subjective things like placement of line breaks leave the choice to humans.